hummingbot/logger/reporting_proxy_handler.py

```diff
@@ -219,11 +219,17 @@ async def send_aggregated_metrics_loop(self, heartbeat_interval_min: float):
                     for exchange in exchanges:
                         markets = set(e["trading_pair"] for e in order_filled if e["event_source"] == exchange)
                         for market in markets:
-                            _, quote = market.split("-")
+                            base, quote = market.split("-")
                             filled_trades = [e for e in order_filled if e["event_source"] == exchange and
                                              e["trading_pair"] == market]
-                            traded_quote_volume = sum(e["price"] * e["amount"] for e in filled_trades)
-                            traded_usdt_value = RateOracle.global_value(quote, traded_quote_volume)
+
+                            if quote == "USDT":
+                                traded_usdt_value = sum(e["price"] * e["amount"] for e in filled_trades)
+                            else:
+                                traded_quote_volume = sum(e["price"] * e["amount"] for e in filled_trades)
+                                quote_usdt_price = await RateOracle.rate_async(f"{quote}-USDT", traded_quote_volume)
```

Comment:
The rate_async will return None, if it can't find the rate for the given pair. So you will have to check for that case.
The return type hint on `rate_async` is not correct (my bad), it should be Optional[Decimal].